scripts/apothecary/formulas/cairo.sh

```diff
@@ -30,40 +30,81 @@ GIT_TAG=$VER
 # download the source code and unpack it into LIB_NAME
 function download() {
 	curl -LO http://cairographics.org/releases/cairo-$VER.tar.xz
-	tar -xf cairo-$VER.tar.xz
-	mv cairo-$VER cairo
-	rm cairo-$VER.tar.xz
-
+	if [ "$TYPE" == "vs" ] ; then
+		7z e cairo-$VER.tar.xz
```

Comment:
Ah, strike that, I installed xz.exe during my research and tar xf must have been calling that.  I'll send a PR with an update.